src/xdocs/checks/whitespace/whitespaceafter.xml

```diff
@@ -122,88 +122,86 @@
       </subsection>
 
       <subsection name="Examples" id="Examples">
-        <p>
+        <p id="Example1-config">
           To configure the check:
         </p>
         <source>
-&lt;module name="WhitespaceAfter"/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;WhitespaceAfter&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
         </source>
-        <p>Example:</p>
-        <div class="wrapper">
-          <pre>
-public void myTest() {
-    if (foo) {              // OK
-        //...
-    } else if(bar) {        // violation
-        //...
+        <source>
+class Example1 {
+  void example() throws Exception {
+    if (true) {
+    } else if(false) { // violation 'not followed by whitespace'
     }
-
-    testMethod(foo, bar);   // OK
-    testMethod(foo,bar);    // violation
-
-    for (;;){}               // OK
-    for(;;){}                // violation, space after 'for' is required
-
-    try (InputStream ignored = System.in) {} // OK
-    try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
-
-    try {} catch (Exception e){} // OK
-    try{} catch (Exception e){} // violation ''try' is not followed by whitespace'
-
-    try {} finally {} // OK
-    try {} finally{} // violation ''finally' is not followed by whitespace'
-
-    try {} catch (Error e){} finally {} // OK
-    try {} catch (Error e){} finally{} // violation ''finally' is not followed by whitespace'
-
-    try {} catch (Exception e){} // OK
-    try {} catch(Exception e){} // violation ''catch' is not followed by whitespace'
-
-    synchronized (this) { } // OK
-    synchronized(this) { } // violation ''synchronized' is not followed by whitespace'
-}
-public String testOne() {
-    return ("a" + "b"); // OK
-}
-public String testTwo() {
-    return("a" + "b"); // violation 'return' is not followed by whitespace'
+    testOne(&quot;x&quot;, &quot;y&quot;);
+    testOne(&quot;z&quot;,&quot;o&quot;); // violation 'not followed by whitespace'
+    for (int i = 0; i &lt; 10; i++){}
+    for(int i = 0; i &lt; 10; i++){} // violation 'not followed by whitespace'
+    try (InputStream ignored = System.in) {}
+    try(InputStream ignored = System.in) {} // violation 'not followed by whitespace'
```

Comment:
Why we missed empty lines? Between blocks .
Let's keep them, they are good.

Reply:
Done. Added lines back.